fsw-react-youtube/src/Components/VideoCard.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import YouTube from 'react-youtube'
+export default function VideoCard() {
+    return (
+        <>
+            <h1>woot</h1>
+            <YouTube videoId={'KQ6zr6kCPj8'}/>
```

Comment:
Looks like you've hardcoded the LMFAO music video 🤣

Here's the most common way to grab the video id dynamically based on the route paramater:

react-router-dom exports a hook called `useParams`. At the top of the video card component, you can grab the value of your route parameter like this:

```
   const { id } = useParams()
```